src/components/Display/Display.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+export default function Display ({ catchphrases }) {
+    
+    return (
+        <>
+            <p>{catchphrases}</p>
```

Comment:
This paragraph tag is causing the phrases to be duplicated both in a list and above mashed together:
![image](https://user-images.githubusercontent.com/74560886/144688406-3ffb8752-95a2-408e-85b3-bbbbf5a9e0d4.png)

Reply:
gross! how'd I miss that!!